packages/docs/stories/components/option.stories.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable arrow-body-style */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable import/no-relative-packages */
+
+import '../../../components/src/components/option/option';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+} from '../../src/helpers/component.js';
+
+const { args: defaultArgs, argTypes } = storybookDefaults('syn-option');
+const { overrideArgs } = storybookHelpers('syn-option');
+const { generateTemplate } = storybookTemplate('syn-option');
+
+const meta: Meta = {
+  args: overrideArgs([
+    { name: 'default', type: 'slot', value: 'Option 1' },
+    { name: 'value', type: 'attribute', value: 'Option 1' },
```

Comment:
We should remove the space here. Otherwise we get this error message in the console: 

![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/96bdc24f-87f6-48c7-ab32-3bea3605be60)
```suggestion
    { name: 'value', type: 'attribute', value: 'Option-1' },
```